Python/pySeaboard.py

```diff
@@ -74,28 +74,28 @@ def noteOn(self, data):
     note = data[1]
     velocity = data[2]
     self.channelNoteArray[channel] = note
-    print "Note On - Ch:" + str(channel) + " Note:" + str(note) + " Velocity:" + str(velocity)
+    print ("Note On - Ch" + str(channel) + " Note:" + str(note) + " Velocity:" + str(velocity))
 
   def noteOff(self, data):
     channel = (data[0] & 0xF) + 1
     note = data[1]
     velocity = data[2]
     self.channelNoteArray[channel] = -1
-    print "Note Off - Ch:" + str(channel) + " Note:" + str(note) + " Velocity:" + str(velocity)
+    print ("Note Off - Ch" + str(channel) + " Note:" + str(note) + " Velocity:" + str(velocity))
 
   def aftertouch(self, data):
     channel = (data[0] & 0xF) + 1
     note = data[1]
     pressure = data[2]
-    print "Aftertouch - Ch:" + str(channel) + " Note:" + str(note) + " Pressure:" + str(pressure)
+    print ("Aftertouch - Ch" + str(channel) + " Note:" + str(note) + " Pressure:" + str(pressure))
 
   def pitchbend(self, data):
     channel = (data[0] & 0xF) + 1
     MSB = data[2]
     LSB = data[1]
     bend = (MSB << 4) + LSB
     note = self.channelNoteArray[channel]
-    print "Pitch Bend - Ch: " + str(channel) + " Note:" + str(note) + " Bend:" + str(bend)
+    print ("Pitch Bend - Ch" + str(channel) + " Note:" + str(note) + " Bend:" + str(bend))
```

Comment:
Looks like we've lot the `:` from these examples. This could be a breaking change if people are parsing this output.